content/en/docs/contribute/intermediate.md

```diff
@@ -324,38 +324,43 @@ graphical Git client instead.
 You only need to clone the repository once per physical system where you work
 on the Kubernetes documentation.
 
-1.  In a terminal window, use `git clone` to clone the repository. You do not
-    need any credentials to clone the repository.
+1.  Create a fork of the `kubernetes/website` repository on GitHub. In your
+    web browser, go to
+    [https://github.com/kubernetes/website](https://github.com/kubernetes/website)
+    and click the **Fork** button. After a few seconds, you are redirected to
+    the URL for your fork, which is `https://github.com/<github_username>/website`.
+
+2.  In a terminal window, use `git clone` to clone the your fork.
 
       ```bash
-      git clone https://github.com/kubernetes/website
+      git clone git@github.com/<github_username>/website
       ```
 
       The new directory `website` is created in your current directory, with
-      the contents of the GitHub repository.
+      the contents of your GitHub repository. Your fork is your `origin`.
 
-2.  Change to the new `website` directory. Rename the default `origin` remote
-    to `upstream`.
+3.  Change to the new `website` directory. Set the `kubernetes/website` repository as the `upstream remote`.
 
       ```bash
       cd website
 
-      git remote rename origin upstream
+      git remote add upstream https://github.com/kubernetes/kubernetes.git
       ```
 
-3.  If you have not done so, create a fork of the repository on GitHub. In your
-    web browser, go to
-    [https://github.com/kubernetes/website](https://github.com/kubernetes/website)
-    and click the **Fork** button. After a few seconds, you are redirected to
-    the URL for your fork, which is typically something like
-    `https://github.com/<username>/website` unless you already had a repository
-    called `website`. Copy this URL.
+4.  Confirm your `origin` and `upstream` repositories
```

Comment:
(minor, optional)
```suggestion
4.  Confirm your `origin` and `upstream` repositories:
```

Reply:
I used a '.' to match the punctuation in the previous steps.